swan-lake/development-tutorials/static-code-analysis/scan-tool.md

```
@@ -180,12 +180,14 @@ You can report the analysis results to platforms such as SonarQube using the `--
 $ bal scan --platforms="sonarqube"
 ```
 
-To specify more than one platform, separate them with commas:
+The `--platforms` option accepts a comma-separated list, so results can be reported to several platforms in a single scan:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)